crates/rolldown_common/src/types/side_effects.rs

```diff
@@ -18,29 +18,31 @@ impl DeterminedSideEffects {
   }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Debug, Clone, Default)]
 /// A field in `package.json`
 pub enum SideEffects {
   Bool(bool),
   String(String),
   Array(Vec<String>),
+  #[default]
+  None,
```

Comment:
Seems misunderstandable. `SideEffects::None` will be misunderstand as  not have side effect. I don't really get the benefit of changing `Option<SideEffects>` to `SideEffects::None`.

Reply:
I don't have a preference, reverted.